.ci/e2e_integration_test/start-e2e.ps1

```diff
@@ -0,0 +1,95 @@
+#
```

Comment:
Please add a note at the top of the file that explains that we are running the Python language worker E2E tests against a prerelease version of the Core Tools (from Integration Tests).